pkg/server/policyrules.go

```diff
@@ -280,7 +280,6 @@ func (ipt *iptableBuffer) renderIngressFrom(s *Server, podInfo *controllers.PodI
 	writeLine(ipt.policyIndex, "-A", fmt.Sprintf("MULTI-%d-INGRESS", pIndex), "-j", chainName)
 
 	s.podMap.Update(s.podChanges)
-	validPeers := 0
```

Comment:
it seems to me that you don't need to count. This could be a boolean and switches to true when there is at least one valid peer

Reply:
I completely removed that variable, as in my idea there is no need to check if there is at least one valid peer. Does it sound good?

Reply:
right, I did not see it correctly